library/httpd/2.4/Dockerfile

```diff
@@ -1,12 +1,29 @@
-FROM httpd:2.4
+FROM ubuntu:16.04
 
-# Adding chameleon specific apache configuration. Some extra mods are now activated:
-# rewrite, proxy, proxy_httpd, proxy_fcgi
-COPY config/httpd.conf /usr/local/apache2/conf/httpd.conf
-COPY config/sites-enabled/vhost.conf /usr/local/apache2/conf/extra/sites-enabled/vhost.conf
+RUN mkdir -p /usr/local/apache2/htdocs/customer/web /usr/local/apache2/conf
+
+RUN apt-get update && apt-get install -y apache2 curl
```

Comment:
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get

"In addition, when you clean up the apt cache by removing /var/lib/apt/lists reduces the image size, since the apt cache is not stored in a layer. Since the RUN statement starts with apt-get update, the package cache is always refreshed prior to apt-get install."

```
RUN apt-get update && apt-get install -y \
    apache2 \
    curl \
  && rm -rf /var/lib/apt/lists/*
```

Reply:
fixed :-)